### PR TITLE
Update VSCode Wizard documentation

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 1]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
 
-## Configure the Project
+## Creating the Project
 
 The **Create New Project** wizard provides two options to start you project:
 
 * **Start from Blank Project** - allows you to create a blank project that is pre-configured for the Progress® Telerik® UI for Blazor components. You can select and add sample pages to get started (a Grid with CRUD operations form a service, a Form, Charts).
 
-* **Start from a Template** - the wizard provides an Admin Dashboard Template - a fully functional sample app that showcases some of the most popular Telerik UI for Blazor components, such as Grid, TileLayout, Form, Charts, Gauges, etc. in a real world scenario.
+* **Start from a Template** - the wizard provides a set of predefined project templates:
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:34]: Double-check the screenshot, it says "Start from Template"

```suggestion
* **Start from Template** - the wizard provides a set of predefined project templates:
```

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 2]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
+    * **CRUD, Form, Chart**&mdash;A small app that showcases the Telerik UI Data Grid, Chart, and forms validation. The Grid also uses a basic CRUD service which is a common pattern for data updates in real applications.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:35]: * Should it be `Form validation` instead?
* For the second sentence, consider going with `The Data Grid also uses a CRUD service (a common pattern for data updates in production-ready applications).`

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 3]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
+    * **Dashboard**&mdash;A basic dashboard layout that uses the TileLayout component for customizable dashboards, and showcases several ways individual blocks (tiles) can fetch data.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:36]:
```suggestion
    * **Dashboard**&mdash;A basic dashboard that showcases the Telerik UI TileLayout component and how individual blocks (tiles) can fetch data.
```

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 4]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
+    * **Admin**&mdash;A small app that shows a dashboard for admin. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:37]:
```suggestion
    * **Admin**&mdash;A small app that showcases a dashboard for administrators. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
```

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 5]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
 
 1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension.
 
-## Configure the Project
+## Creating the Project
 
 The **Create New Project** wizard provides two options to start you project:
 
 * **Start from Blank Project** - allows you to create a blank project that is pre-configured for the Progress® Telerik® UI for Blazor components. You can select and add sample pages to get started (a Grid with CRUD operations form a service, a Form, Charts).
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:32]: In the screenshot, it's `Start from Blank App`.

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 6]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
+* **Start from a Template** - the wizard provides a set of predefined project templates:
+    * **CRUD, Form, Chart**&mdash;A small app that showcases the Telerik UI Data Grid, Chart, and forms validation. The Grid also uses a basic CRUD service which is a common pattern for data updates in real applications.
+    * **Dashboard**&mdash;A basic dashboard layout that uses the TileLayout component for customizable dashboards, and showcases several ways individual blocks (tiles) can fetch data.
+    * **Admin**&mdash;A small app that shows a dashboard for admin. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
 
->caption The wizard allows you to configure the following options:
+## Configuring the Project
+
+The wizard allows you to configure the following options:
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:41]: Is this for both options (blank app and template), including all template sub-options? If this is the case, consider going with `In the **Telerik UI for Blazor Template Wizard**, you can configure the following options for your new Blazor app:**

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 7]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
 
 * Location and name for your app.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:43]: Go with what you're seeing in the screenshot. In this case, it seems to be `Project name and output path.` If these are not native options to other VSCode wizards, consider providing more detail.

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 8]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
-* Your Telerik license (trial or commercial, depending on what you have).
-* A WebAssembly or Server-Side Blazor app.
+* Hosting model type - [WebApp]({%slug getting-started/web-app%}), [WebAssembly]({%slug getting-started/client-side%}), [Server-Side]({%slug getting-started/server-side%}) or [Hybrid Blazor MAUI]({%slug getting-started/hybrid-blazor%}) app.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:44]: Go with what people will see in the screenshot. The current screenshot says WebApp, Client, Server, and Hybrid. Provide additional details in parenthesis. Also, the names of UI options, menus, etc. are usually formatted with bold. For example:
```suggestion
* Hosting model type&mdash;[**WebApp**]({%slug getting-started/web-app%}), [**Client** (also: WebAssembly)]({%slug getting-started/client-side%}), [**Server**]({%slug getting-started/server-side%}), or [**Hybrid** (a hybrid Blazor MAUI)]({%slug getting-started/hybrid-blazor%}) app.
```

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 9]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
+* Your Telerik license (trial or commercial, depending on what you have).
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:45]: I would use the opportunity to briefly address paid vs trial. Do they need to set up their license somehow, do they need to do anything special if they choose paid, etc. For example: 
```suggestion
* Your Telerik license. If set to **Trial**, you will see a trial watermark and banner over your application. If set to **Paid**, you need to ... 
```

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 10]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
+
+The wizard allows you to configure the following options:
 
 * Location and name for your app.
-* Your Telerik license (trial or commercial, depending on what you have).
-* A WebAssembly or Server-Side Blazor app.
+* Hosting model type - [WebApp]({%slug getting-started/web-app%}), [WebAssembly]({%slug getting-started/client-side%}), [Server-Side]({%slug getting-started/server-side%}) or [Hybrid Blazor MAUI]({%slug getting-started/hybrid-blazor%}) app.
+* Your Telerik license (trial or commercial, depending on what you have). 
+* Whether to use localization in the project or not.
+* Whether to use font icons in the project or not.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:47]: In a future version, consider reworking the UI for this. The Icon Settings label for the button is not intuitive or user-friendly. I would probably switch the labels: `Configure icons` with the button label being `Use font icons`.

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 11]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
 * The .NET version you want to use.
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:48]: Can people choose a different .NET version? Does it depend on what they have installed on their system? If the latter, I would add a line saying `The wizard shows only the .NET versions available on your system.`

File: getting-started/vs-code-integration/new-project-wizard.md [hunk 12]
```diff
@@ -25,21 +25,27 @@ To create a Telerik-enabled Blazor project:
...
-* The Telerik UI for Blazor version.
 * Sample pages that you can add to get started (available for `Start from Blank Project` option).
```
Comment (ikoevska) [inline getting-started/vs-code-integration/new-project-wizard.md:49]:
```suggestion
* Sample pages that you can add to get started (available when `Start from Blank App` is selected).
```

**Review by ikoevska — COMMENTED** [review]
A general note when listing UI options and explaining them in text - always use precisely what's in the screenshot.

File: getting-started/vs-code-integration/new-project-wizard.md (hunk 11 above)
Comment (ntacheva) [reply getting-started/vs-code-integration/new-project-wizard.md:48]: The wizard provides a general .NET version configuration option regardless of what is installed on one's machine.

File: getting-started/vs-code-integration/new-project-wizard.md (hunk 10 above)
Comment (ntacheva) [reply getting-started/vs-code-integration/new-project-wizard.md:47]: This depends on the team that manages the VSCode extension. Maybe you can open an issue with the suggestion or discuss it with a member of the team.

**Review by dimodi — APPROVED** [review]

File: getting-started/vs-code-integration/new-project-wizard.md (hunk 11 above)
Comment (ikoevska) [reply getting-started/vs-code-integration/new-project-wizard.md:48]: If that's the case, I'd argue about the merit of a screenshot showing how people can choose from one possible option.

**Pull request merged**